ecl/data_source_ecl_network_fic_gateway_v2.go

```diff
@@ -0,0 +1,128 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/nttcom/eclcloud/ecl/network/v2/fic_gateways"
+)
+
+func dataSourceNetworkFICGatewayV2() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNetworkFICGatewayV2Read,
+
+		Schema: map[string]*schema.Schema{
+			"region": &schema.Schema{
```

Comment:
以下のように型は省略できます。他のattributeも含めて修正していただけますか？
```suggestion
			"region": {
```